packages/babel-parser/src/parser/comments.ts

```diff
@@ -237,16 +206,15 @@ export default class CommentsParser extends BaseParser {
    * method later. In this case the identifier is not part of the AST and we
    * should sync the knowledge to commentStacks
    *
-   * For example, when parsing */
-  // async /* 1 */ function f() {}
-  /*
+   * ↓↓↓ We had escaped "\/" to "\\/" for comments
+   * For example, when parsing *\/
+   * // async /* 1 *\/ function f() {}
```

Comment:
I actually prefer the old comment style, so that it didn't need escapes :stuck_out_tongue:

Reply:
Can you try if https://github.com/JLHwung/babel/commit/6eee6d0be2bff890a1f0a7ad6384d56f6aecf551 works? I added U+200B between `*` and `/`.

Reply:
Wow this is awesome, thank you!